examples/history_vehicles_replacement_for_imitation_learning.py

```diff
@@ -14,69 +18,164 @@
 
 
 class KeepLaneAgent(Agent):
-    def __init__(self, target_speed=15.0):
+    def __init__(self, target_speed: float = 15.0):
         self._target_speed = target_speed
 
-    def act(self, obs):
+    def act(self, obs) -> Tuple[float, int]:
```

Comment:
from smarts.core.sensors import Observation
```suggestion
    def act(self, obs: Observation) -> Tuple[float, int]:
```